src/screens/LoginScreen.js

```diff
@@ -55,105 +51,99 @@ class LoginScreen extends Component {
-                                label={login.password}
-                                placeholder={login.password_placeholder}
-                                value={password}
-                                onChangeText={this.props.passwordChanged}
-                                onPress={() => this.setState({hidden: !hidden})}
-                                hidden={this.state.hidden}
-                            />
-
-                            <Button
-                                onPress={this.onLoginPress}
-                                large
-                                buttonStyle={buttonStyle}
-                                icon={{name: 'envira', type: 'font-awesome'}}
-                                title='Sign in'
-                                rounded={true}
-                            >
-                                {login.login}
-                            </Button>
-
-                          </CardSection>
-                    </View>
+            <View style={containerStyle}>
+                <View style={topContainer}>
+                    <Card
+                        style={heroContainer}
+                        title={_.upperCase(campo_libre)}
+                        image={require('../../assets/hero.jpg')}
+                    >
+                        <Text>
+                            {tagline}
+                        </Text>
+                    </Card>
+                </View>
 
+                <View style={buttonContainer}>
                     <Button
-                       large
-                       transparent
-                       icon={{name: 'facebook', type: 'font-awesome'}}
-                       title='Sign in with Facebook'
-                       buttonStyle={facebookStyle}
-                       onPress={this.onPressFacebookLogin}
-                       rounded={true}
-                    >
-                     {login.login_with_facebook}
-                    </Button>
-                 </View>
-            </Card>
+                        large
+                        transparent
+                        icon={{name: 'facebook', type: 'font-awesome'}}
+                        title={login_with_facebook}
+                        buttonStyle={facebookStyle}
+                        onPress={this.onPressFacebookLogin}
+                        rounded={true}
+                    />
+
+                    <Button
+                        onPress={this.onPressContinueAsGuest}
+                        large
+                        buttonStyle={buttonStyle}
+                        icon={{name: 'envira', type: 'font-awesome'}}
+                        title={login_as_guest}
+                        rounded={true}
+                    />
+                </View>
+            </View>
+        );
+    }
+
+    render() {
+        const {fillScreen} = styles;
+
+        return (
+            <View style={fillScreen}>
+                {this.renderPage()}
```

Comment:
Now calling a conditional render function instead of dropping the JSX straight in here. I suspect we'll be doing this more and more in our other screens.